apis/apis.json

```diff
@@ -473,7 +473,7 @@
       "Google.Apis.Storage.v1": "1.30.0.1014"
     },
     "testDependencies": {
-      "Google.Cloud.Iam.V1": "project",
+      "Google.Cloud.Iam.V1": "1.0.0-beta13",
```

Comment:
beta15 is the latest

Reply:
Nope, not for IAM - PubSub is beta15, but this line is about IAM :)